template/src/pageComponents/mint/ContractDemo.tsx

```diff
@@ -68,7 +94,17 @@ export default function MintContractDemo() {
       <div className="flex-shrink-1 mt-10 w-full flex-grow-0 lg:mt-0">
         <h1 className="text-4xl font-bold">{collectionName}</h1>
 
-        <h2 className="my-5">{String(ethAmount)} ETH</h2>
+        <h2 className="my-5">
+          Estimated tx fee:{' '}
+          {isLoadingFeeEstimate ? (
+            <SpinnerIcon className="inline animate-spin" height="1.2rem" width="1.2rem" />
+          ) : (
+            <>
+              {mintTxFeeEstimation}
```

Comment:
Hmm, I think we should also add `0.001 ETH` that is required to mint an NFT and display the estimated total cost, what are your thoughts on that?

Reply:
is 0.001 a requirement? I couldn't see it in a CustomERC1155 contract, and it's not verified so couldn't check in basescan. I assumed this is a free mint so hence I opted for displaying transaction fee instead of mint price. I will try to get the mint price later, but for some reason I thought I tried and couldn't see it

Reply:
I think you are right, seems like the `ethAmount` here is was incorrectly added and this is actually a free mint. Nice catch!

Reply:
I'll add a comment, so when PaidMint example is added, devs don't forget to add a mint price there

Reply:
Awesome! Thank you